code/game/objects/items/weapons/explosives.dm

```diff
@@ -126,4 +126,4 @@
 				blast_flash = blast_light + round(new_blast_power * 0.75)
 			else
 				to_chat(user, "<span class='notice'>The [I] is not any better than the component already installed into this charge!</span>")
-	return .
\ No newline at end of file
+	return .

```

Comment:
```suggestion
```
This is the curse of me reviewing PRs that touch a hundred somewhat-random files.

Reply:
`return .` is actually the /tg/ favored way of implying returning the default return value (even though return does that anyway). Lummox updated byond to remove an extra instruction just so this pattern wasn't detrimental.

Although I think it's stylistically detrimental. 🥴